### PR TITLE
WRP-7678: Upgrade `less-loader` dependency to latest version

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Less-loader needs to be in sync with the version enact cli is using.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated less-loader dependency to v11.
There were no breaking changes affecting our code https://github.com/webpack-contrib/less-loader/releases.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Minimum supported Node.js version is now 14.15.0 (https://github.com/webpack-contrib/less-loader/releases/tag/v11.0.0).
API change from release v10.0.0 (https://github.com/webpack-contrib/less-loader/releases/tag/v10.0.0) does not affect our code.

### Links
[//]: # (Related issues, references)
WRP-7678

### Comments

Enact-DCO-1.0-Signed-off-by: Stanca Pop (stanca.pop@lgepartner.com)

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by ion-andrusciac-lgp — APPROVED** [review]
LGTM

**Comment by hong6316** [comment]
In `less-loader` v10.0.0, there an API change exists: https://github.com/webpack-contrib/less-loader/releases/tag/v10.0.0
Is it possible to check if this API change does not affect the `storybook-utils`?
I'll merge it as soon as it's checked.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Updated `less-loader` version to `^11.1.0`.
```
Comment (hong6316) [inline CHANGELOG.md:3]:
```suggestion
* Updated `less-loader` version to `^11.1.0`.
* Updated the minimum version of Node to `14.15.0`.
```

**Comment by stanca-pop-lgp** [comment]
> In `less-loader` v10.0.0, there an API change exists: https://github.com/webpack-contrib/less-loader/releases/tag/v10.0.0 Is it possible to check if this API change does not affect the `storybook-utils`? Please answer this question and describe on `Additional Considerations` in the PR description.
> 
> I'll merge it as soon as it's checked.

API change from release v10.0.0 does not affect the `storybook-utils`.

**Pull request merged**